governance-exercises/CivicSage Exercises/Ex01/3.2 Increase Spork DAI Deposit Rate - JULY 24, 2023.md

```diff
@@ -0,0 +1,51 @@
+---
+title: Increase Spork DAI Deposit Rate - JULY 24, 2023
+summary: Signal your support or opposition to increase of DAI deposit rate on Spork from 2% to 2.5%.
```

Comment:
Use Grammarly to avoid grammatical errors
```suggestion
summary: Signal your support or opposition to the increase of the DAI deposit rate on Spork from 2% to 2.5%.
```